woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php

```diff
@@ -123,6 +125,19 @@ public function setup_configs() {
 			$this->public_key = $this->is_sandbox_mode ? $this->configs->settings['sandbox_public_key'] : $this->configs->settings['live_public_key'];
 		}
 
+		public function check_sandbox_mode() {
```

Comment:
I think this method can be more generically to support future messages. What do you think?